lib/teleterm/api/proto/v1/service.proto

```diff
@@ -126,6 +147,69 @@ enum PasswordlessPrompt {
   PASSWORDLESS_PROMPT_CREDENTIAL = 3;
 }
 
+message GetAccessRequestRequest {
+  // cluster_uri is the cluster uri
```

Comment:
I don't know that this comment is adding much value. Up to you, but my vote would be to simply remove it if we don't have anything useful to add.